adr/0022-integration-quality-scale.md

```diff
@@ -0,0 +1,169 @@
+- Provides everything “Silver” has.
+- Has the best end-user experience an integration can offer; streamlined and intuitive.
+- Can be automatically discovered, simplifying the integration setup.
+- Integration can be reconfigured and adjusted.
+- Supports translations.
+- Extensive documentation, aimed at non-technical users.
+- It supports updating the software/firmware of devices through Home Assistant when possible.
+- Provides the tooling and documentation troubleshooting issues.
+- The integration has automated tests covering the entire integration.
+- Required level for integrations providing devices in the Works with Home Assistant program.
+
+#### 🏆 Platinum
+
+The platinum tier has the following characteristics:
+- Provides everything “Gold” has.
+- All source code follows all coding and Home Assistant integration standards and best practices and is fully typed with type annotations and clear code comments for better code clarity and maintenance.
+- A fully asynchronous integration code base ensures efficient operation.
+- Implements efficient data handling, reducing network and CPU usage.
+
+### Special tiers
+
+Besides the four scaled tiers mentioned above, Home Assistant also provides four special tiers to denote a special case, causing them not to fit into the regular scale set out above.
+
+Note that while an integration might have been given one of these special tiers, they still might pursue the integration quality equivalent to any of the other scaled tiers.
+
+#### ❓ No score
+These integrations can be set up through the Home Assistant user interface. The “No score” designation doesn’t imply that they are bad or buggy, instead, it indicates that they haven’t been assessed according to the quality scale or that they need some maintenance to reach the now-considered minimum “Bronze” standard.
+
+The “No score” tier cannot be assigned to new integrations, as they are required to have at least a “Bronze” level when introduced. The Home Assistant project encourages the community to help updating these integrations without a score to meet at least the “Bronze” level requirements.
+
+Characteristics:
+- Not yet scored or lacks sufficient information for scoring.
+- Can be set up via the UI, but may need enhancements for a better experience.
+- May function correctly, but hasn’t been verified against current standards.
+- Documentation most often provides only basic setup steps.
+
+#### 🏠 Internal
+This tier is assigned to integrations used internally by Home Assistant. These integrations provide basic components and building blocks for Home Assistant's core program or for other integrations to build on top of it.
+
+Everything in Home Assistant is extendable using integrations. In fact, Home Assistant uses its own integration logic to extend base functionality. For example, the automation engine of Home Assistant is an integration. Those internal parts are thus marked with this “Internal” tier.
```

Comment:
I'd remove this paragraph. It'd not needed. If wanted put the example in the first paragraph of this section instead.